### PR TITLE
added inspection list screen. resolves #113

### LINKED ISSUE #113: Adding inspection list screen

### PR DESCRIPTION
**Pull request by SimonSchwierzeck**

### CODE REVIEW

**Comment by SimonOyen** [comment]
PR blocked until #86 is closed.

File: lib/blocs/inspectiondata_bloc/inspectiondata_bloc.dart [hunk 1]
```diff
@@ -35,5 +45,50 @@ class InspectionDataBloc
       print(inspectionList.length);
       yield InspectionReadyState();
     }
+    if (event is GetAllInspectionPreviewEvent) {
+      if (this.state is! InitialInspectionDataState) {
+        yield InspectionChangingState();
+      }
+      inspectionList = await _inspectionRepo.getAllInspectionPreviews();
+      print(inspectionList.length);
+      yield InspectionReadyState();
```
Comment (SimonOyen) [inline lib/blocs/inspectiondata_bloc/inspectiondata_bloc.dart:58]: Has to be changed to InspectionPreviewByNestingBoxId when #86 is resolved

**Review by SimonOyen — CHANGES_REQUESTED** [review]
Please look into the problem with files in the docs/ directory and implement the InspectionByNestingBoxId functionality.

**Comment by SimonSchwierzeck** [comment]
Please dont break other peoples repositorys.

**Review by SimonOyen — APPROVED** [review]
:medal_sports:

**Pull request merged**